docs/release-notes/5.7.39-31.61.md

```diff
@@ -0,0 +1,29 @@
+# Percona XtraDB Cluster 5.7.39-31.61 (2022-08-29)
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and the security features your business requires to meet your customer expectations and business goals.
+
+## Release Highlights
+
+The improvements and bug fixes for MySQL 5.7.39, provided by Oracle, and included in Percona Server for MySQL are the following:
```

Comment:
We only seem to point out the changes from upstream Oracle MySQL 5.7.39 here.
Since PXC is based on Percona Server for MySQL, shouldn't we also point to [our own release notes](https://docs.percona.com/percona-server/5.7/release-notes/Percona-Server-5.7.39-42.html) here?

Reply:
OK. Will add a link.